pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -4899,9 +4899,7 @@
       ],
       "BaselineVersion": "4.4.0",
       "InboxOn": {
-        "netcoreapp2.0": "4.1.1.0",
```

Comment:
This library is still inbox on these platforms so these should remain.

Reply:
You mean this specific version of that library right? So on netcoreapp2.0 when using 4.2.0.0 we will get newer bits overriding the shared fx yeah?

Reply:
Yes that is correct.

Reply:
Exactly, never remove anything from the package index for past releases.

Reply:
I still want to remove the line for 2.1, though, right?

Reply:
>  remove the line for 2.1

You'll want to update the line for netcoreapp2.1 to be version 4.2.0.0.

Reply:
Yup, thanks, I realized that after I wrote my comment.